src/driver/express/ExpressDriver.ts

```diff
@@ -58,17 +58,19 @@ export class ExpressDriver extends BaseDriver {
     let middlewareWrapper;
 
     // if its an error handler then register it with proper signature in express
-    if ((middleware.instance as ExpressErrorMiddlewareInterface).error) {
+    if (middleware.target.prototype && middleware.target.prototype.error) {
```

Comment:
You can simplify expression as `if (middleware.target?.prototype?.error)`. Same for others here.